.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules
 /dist
-/.idea
\ No newline at end of file
+/.idea
+/src/img
```

Comment:
Why aren't images supposed to be in repo?

Reply:
It is template images, after DB connect they will adds in admin panel into db(or smth) and will be taken from it.
package is empty
Moreover i still don't fix local images trouble (shame) and will return in need...

Reply:
Should i mark this comment as resolved?